src/main/kotlin/week1/racingcar/InputView.kt

```diff
@@ -0,0 +1,15 @@
+package week1.racingcar
+
+class InputView {
+    fun startInput(): InputResult {
+        println("자동차 대수는 몇 대인가요?")
+        val carNumber = readlnOrNull()?.toInt()
+        println("시도할 횟수는 몇 회인가요?")
+        val timesToRunGame = readlnOrNull()?.toInt()
+
+        require(carNumber != null)
+        require(timesToRunGame != null)
```

Comment:
`require` 대신에 `requireNotNull`을 활용해보면 어떨까요? 😃
또한 적절하게 예외 메시지를 추가해서 왜 예외가 발생했는지를 한 눈에 파악할 수 있도록 해보면 좋을 것 같아요. 😉

또한 입력을 받은시점과 검증을 하는 부분이 조금 떨어져있는 것 같은데, 
가급적 입력을 받고 가까운 시점에 검증을 하는 것이 더 좋을 것 같습니다. 🤔

```suggestion
        println("자동차 대수는 몇 대인가요?")
        val carNumber = readlnOrNull()?.toInt()
        requireNotNull(carNumber) { "자동차 갯수는 null을 허용하지 않습니다." }

        println("시도할 횟수는 몇 회인가요?")
        val timesToRunGame = readlnOrNull()?.toInt()
        requireNotNull(timesToRunGame) { "레이스 횟수는 null을 허용하지 않습니다." }
```